src/app/functions/server/getSession.ts

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { getIronSession, IronSession } from "iron-session";
+import { NextResponse } from "next/server";
+import { UserData } from "../../transitionTypes";
+
+export type SessionData = Partial<{
+  oauthState: string;
+  user: UserData;
+}>;
+
+const sessionConfig = {
+  cookieName: "monitor-iron-session",
+  password: process.env.IRON_SESSION_PASSPHRASE!,
```

Comment:
AIUI this is used to encrypt the client-side cookie. I wonder if we could re-use the existing env var `COOKIE_SECRET` for this?

Reply:
I was thinking about the same thing.. Yes I don't see a reason why not